sync/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
     implementation project(':services:serviceutils')
     implementation project(':ssz')
     implementation project(':storage')
```

Comment:
nit: I wonder if we can remove a bunch of these dependencies on storage now and only depend on storage:api?

Reply:
probably if they're not using storage/server... i'll take a quick look before i merge

Reply:
we'll need to break out storage:server and storage:client, then we can remove a lot of the dependencies, and also the remaining api member in the main server folder could be moved after that.